tests/src/test/java/alluxio/master/file/FileSystemMasterIntegrationTest.java

```diff
@@ -949,6 +955,10 @@ public void exec(int depth, int concurrencyDepth, AlluxioURI path) throws Except
     }
   }
 
+  /**
+   * A class to start a thread to create file, complete file and delete file continuity, until the thread is
```

Comment:
Replace the description with `A class to start a thread that creates a file, completes the file and then deletes the file.`